docs/setup/setup.md

```diff
@@ -153,6 +153,12 @@ openssl req -new -key kubeedge.key -out kubeedge.csr
 openssl x509 -req -in kubeedge.csr -CA rootCA.crt -CAkey rootCA.key -CAcreateserial -out kubeedge.crt -days 500 -sha256 
 ```
 ## Build  
+**Info**
+If your are running go 1.12 or higher make sure to disable go mod and set GOPATH.
+```shell
+export GO111MODULE=off
```

Comment:
Does building or running KubeEdge cause a problem if go mod is not disabled ? I haven't tried with latest go versions.

Reply:
It does for me. It causes this issue:
go: github.com/Sirupsen/logrus@v1.4.1: parsing go.mod: unexpected module path "github.com/sirupsen/logrus"
similar for go-chassis dep